### PR TITLE
Draft WN post for RSS functionality

### PR DESCRIPTION
**Pull request by austin-schaefer**
### Link to page in Gatsby preview

> https://docswebsitedevelop-whatsnewrssisnew.gtsb.io/whats-new/2021/05/rss/

### Give us some context

* This PR is a small What's New post to tell users we now have RSS feeds for What's New and security bulletins
* Currently set "release date" to 5/21 so this can slip in ahead of FutureStack, but this can go anytime @daynaslord is okay with it launching.
* @austin-schaefer wrote the copy, need someone from PMM to check it out for WN voice/tone (I went pretty loose and informal with it, wondering if I went too far)

### Reviewers:

* Need a TW review for voice/tone/local build quality
* Need review from @daynaslord or delegate for PMM perspective

### Other notes:

* I decided to skip a video/image for this one, couldn't think of a particularly compelling visual for RSS
* I tested this locally, but would still appreciate the TW reviewer building locally in case I messed something up (this is my first time working with What's New content)

### CODE REVIEW

File: src/content/whats-new/2021/05/rss.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "RSS for docs so you can automate all the Slack bots"
+summary: "Keep track of what's new with RSS feeds for What's New (and security bulletins)"
+releaseDate: '2021-05-21'
+---
+Have you heard? RSS is back! Our docs have long included RSS feeds for each of our agents. Lately we've seen lots of requests to add RSS for What's New and Security Bulletins, so we've done just that! Now you can subscribe to all the newness New Relic has to offer.
```
Comment (urbiz-nr) [inline src/content/whats-new/2021/05/rss.md:6]: Really minor, maybe write "our Security Bulletins"

File: src/content/whats-new/2021/05/rss.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+RSS is great in your feed reader, of course, but it also makes it easy to automatically keep up with New Relic in other places. For example, [Slack's built in RSS app](https://slack.com/help/articles/218688467-Add-RSS-feeds-to-Slack) can automatically post to a channel whenever a new agent release drops, or you could use Zapier for all manner of RSS-based automation. (Want to turn your smart lightbulbs green every time we release a new Node.js agent? Have at it!)
```
Comment (urbiz-nr) [inline src/content/whats-new/2021/05/rss.md:8]: That final "Have at it!" is a little weird. How about "Go for it!"'?

**Review by urbiz-nr — APPROVED** [review]
Reads great!

File: src/content/whats-new/2021/05/rss.md (hunk 2 above)
Comment (austin-schaefer) [reply src/content/whats-new/2021/05/rss.md]: Good call!

File: src/content/whats-new/2021/05/rss.md (hunk 1 above)
Comment (austin-schaefer) [reply src/content/whats-new/2021/05/rss.md]: 👍

**Review by paperclypse — APPROVED** [review]
Looks good!

**Pull request merged**